enterprise/server-enterprise/src/test/java/org/neo4j/server/enterprise/StandaloneClusterClientIT.java

```diff
@@ -299,8 +299,7 @@ private boolean startStandaloneClusterClient( File configFile, Map<String, Strin
     private Process startStandaloneClusterClientProcess( File configFile, Map<String, String> config )
             throws Exception
     {
-        List<String> args = new ArrayList<>( asList( "java", "-cp", getProperty( "java.class.path" ),
-                "-Dorg.neo4j.cluster.logdirectory=" + directory.getAbsolutePath() ) );
+        List<String> args = new ArrayList<>( asList( "java", "-cp", getProperty( "java.class.path" ) ) );
```

Comment:
Don't you need to configure `dbms.directories.logs` to avoid writing logs into some random directory based on the CWD and default setting?